lib/logflare/sql.ex

```diff
@@ -111,6 +111,10 @@ defmodule Logflare.Sql do
         ast: statements
       })
       |> Parser.to_string()
+    else
+      error ->
+        Logger.warning("SQL transformation failed", error_string: query)
```

Comment:
these errors shouldn't be logged as user error would be high (parsing on each endpoint query change). we would want to log the query on an error and reraise it, something like that.

Reply:
might be even better to set the Logger.metadata for a `:query_string` field so that any related error log raised gets it